app/controllers/shopify_app/sessions_controller.rb

```diff
@@ -1,5 +1,5 @@
 module ShopifyApp
-  class SessionsController < ApplicationController
+  class SessionsController < ActionController::Base
     include ShopifyApp::SessionsConcern
```

Comment:
Actually, never mind. I just realized that `LoginProtection` is included in `SessionsConcern`